packages/lodestar-validator/src/api/impl/rest/validator/validator.ts

```diff
@@ -12,23 +13,25 @@ import {
   CommitteeIndex,
   Epoch,
   ProposerDuty,
+  SignedAggregateAndProof,
   SignedBeaconBlock,
   Slot,
-  SignedAggregateAndProof,
+  ValidatorIndex,
 } from "@chainsafe/lodestar-types";
-import {IValidatorApi} from "../../../interface/validators";
-import {HttpClient, urlJoin} from "../../../../util";
 import {ILogger} from "@chainsafe/lodestar-utils";
-import {IBeaconConfig} from "@chainsafe/lodestar-config";
 import {Json, toHexString} from "@chainsafe/ssz";
+import {HttpClient, urlJoin} from "../../../../util";
+import {IValidatorApi} from "../../../interface/validators";
 
 export class RestValidatorApi implements IValidatorApi {
   private readonly client: HttpClient;
+  private readonly clientV2: HttpClient;
 
   private readonly config: IBeaconConfig;
 
   public constructor(config: IBeaconConfig, restUrl: string, logger: ILogger) {
     this.client = new HttpClient({urlPrefix: urlJoin(restUrl, "validator")}, {logger});
+    this.clientV2 = new HttpClient({urlPrefix: urlJoin(restUrl, "/eth/v1/validator")}, {logger});
```

Comment:
clientV2 = /eth/v1 ?

Would it be more clear to do clientV1 = /eth/v1

Reply:
When I'm done it will be just "client", this is just so I don't have to update everything at once